ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -102,17 +103,58 @@ for imem in $(seq 1 "${NMEM_REGRID}"); do
         MEMDIR=${memchar} YMD=${PDY} HH=${cyc} declare_from_tmpl \
             COMIN_SOIL_ANALYSIS_MEM:COM_ATMOS_ANALYSIS_TMPL
     fi
- 
+    
+    HIDE_LIAU="!"
+    rm -f "regrid.nml"
+    atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
+
     for FHR in "${soilinc_fhrs[@]}"; do
         ${NCP} "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci00${FHR}.nc" \
-               "${DATA}/enkfgdas.sfci.nc"
+               "${DATA}/${in_fname}"
 
         ${APRUN_REGRID} "${REGRID_EXEC}" "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
 
         for n in $(seq 1 "${ntiles}"); do
-            cpfs "${DATA}/sfci.tile${n}.nc"  "${COMOUT_ATMOS_ANALYSIS_MEM}/sfci00${FHR}.tile${n}.nc" 
+            cpfs "${DATA}/${out_fname}.tile${n}.nc"  "${COMOUT_ATMOS_ANALYSIS_MEM}/${out_fname}00${FHR}.tile${n}.nc"
         done
     done
+
+    if [[ ${DO_LAND_IAU} = ".true." ]]; then 
+        
+        HIDE_LIAU=" "
+        rm -f "regrid.nml"
+        atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
+        
+	      #fix until reg code time dim issues are sorted out. TODO: time dim in regr/ufs code
+        if [[ "${n_tims}" -eq 1 ]]; then
+            for FHI in "${ifhrsi[@]}"; do
+                ${NCP} "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci0${FHI}.nc" \
+                       "${DATA}/${in_fname}"
+            done
+        else
+            for FHI in "${ifhrsi[@]}"; do
+	              ${NCP} "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci0${FHI}.nc" \
+                       "${DATA}/${in_fname}.${FHI}"
+            done
+        fi
+        
+	      ${APRUN_REGRID} "${REGRID_EXEC}" "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
+   
+        #fix until reg code time dim issues are sorted out. TODO: time var in regr/ufs code
+	      if [[ "${n_tims}" -eq 1 ]]; then 
+            for n in $(seq 1 "${ntiles}"); do
+                ncecat -O -u Time "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"   
+                ncap2 -A -s @all="{${ifhrsf[*]}}" "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"
+                ncap2 -O -s'Time[Time]=@all' "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"
```

Comment:
I would recommend trapping errors if they occur
```suggestion
                ncecat -O -u Time "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"
                export err=$?; err_chk
                ncap2 -A -s @all="{${ifhrsf[*]}}" "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"
                export err=$?; err_chk
                ncap2 -O -s'Time[Time]=@all' "${out_fname}.tile${n}.nc" "${out_fname}.tile${n}.nc"
                export err=$?; err_chk
```